<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1478</width>
    <height>996</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>9</y>
     <width>1261</width>
     <height>841</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1700</width>
     <height>1000</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pgeMain">
    <widget class="QPushButton" name="btnSnake">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:1, y1:0.653, x2:1, y2:0, stop:0 rgba(55,55,55, 255), stop:0.798883 rgba(110, 58, 58, 255), stop:1 rgba(255, 0, 0, 255));	
	border-style: outset;
	border-radius: 25px;
	background-image: url(:/MainWindow/snake.png);
}
QPushButton:hover{
	background-image: url(:/MainWindow/snake.png);
	background-color: rgba(0, 0, 0, 128);
}</string>
     </property>
     <property name="text">
      <string>Snake</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLED">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:1, y1:0.653, x2:1, y2:0, stop:0 rgba(55,55,55, 255), stop:0.798883 rgba(110, 58, 58, 255), stop:1 rgba(255, 0, 0, 255));
border-style: outset;
border-radius: 25px;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:1, y1:0.653, x2:1, y2:0, stop:0 rgba(96, 96, 106, 255), stop:0.798883 rgba(110, 58, 58, 255), stop:1 rgba(255, 0, 0, 255));
}</string>
     </property>
     <property name="text">
      <string>LED</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSimon">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(55, 55, 55);
border-style: outset;
border-radius: 25px;
border-image: url(:/MainWindow/simon.png);
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:1, y1:0.653, x2:1, y2:0, stop:0 rgba(96, 96, 106, 255), stop:0.798883 rgba(110, 58, 58, 255), stop:1 rgba(255, 0, 0, 255));
}</string>
     </property>
     <property name="text">
      <string>Simon</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAccel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:1, y1:0.653, x2:1, y2:0, stop:0 rgba(55,55,55, 255), stop:0.798883 rgba(110, 58, 58, 255), stop:1 rgba(255, 0, 0, 255));
border-style: outset;
border-radius: 25px;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:1, y1:0.653, x2:1, y2:0, stop:0 rgba(96, 96, 106, 255), stop:0.798883 rgba(110, 58, 58, 255), stop:1 rgba(255, 0, 0, 255));
}</string>
     </property>
     <property name="text">
      <string>Accel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPoten">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:1, y1:0.653, x2:1, y2:0, stop:0 rgba(55,55,55, 255), stop:0.798883 rgba(110, 58, 58, 255), stop:1 rgba(255, 0, 0, 255));
border-style: outset;
border-radius: 25px;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:1, y1:0.653, x2:1, y2:0, stop:0 rgba(96, 96, 106, 255), stop:0.798883 rgba(110, 58, 58, 255), stop:1 rgba(255, 0, 0, 255));
}</string>
     </property>
     <property name="text">
      <string>Poten</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDebug">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>210</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:1, y1:0.653, x2:1, y2:0, stop:0 rgba(55,55,55, 255), stop:0.798883 rgba(110, 58, 58, 255), stop:1 rgba(255, 0, 0, 255));
border-style: outset;
border-radius: 25px;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:1, y1:0.653, x2:1, y2:0, stop:0 rgba(96, 96, 106, 255), stop:0.798883 rgba(110, 58, 58, 255), stop:1 rgba(255, 0, 0, 255));
}</string>
     </property>
     <property name="text">
      <string>idk</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConfiguration">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>110</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:1, y1:0.653, x2:1, y2:0, stop:0 rgba(55,55,55, 255), stop:0.798883 rgba(110, 58, 58, 255), stop:1 rgba(255, 0, 0, 255));
border-style: outset;
border-radius: 25px;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:1, y1:0.653, x2:1, y2:0, stop:0 rgba(96, 96, 106, 255), stop:0.798883 rgba(110, 58, 58, 255), stop:1 rgba(255, 0, 0, 255));
}</string>
     </property>
     <property name="text">
      <string>Configuration</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pgeSnake"/>
   <widget class="QWidget" name="pgeAccel"/>
   <widget class="QWidget" name="pgeLED">
    <widget class="QLabel" name="labInstruc">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>600</width>
       <height>300</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labResult">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>350</y>
       <width>600</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pgeSimon"/>
   <widget class="QWidget" name="pgePoten">
    <widget class="QLabel" name="labPotSequence">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>301</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_7">
    <widget class="QLabel" name="labDebug_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>381</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLCDNumber" name="lcdClock">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="prbBargraph">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>41</width>
     <height>511</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labDebug">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="btnHome">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
	background-color: rgb(96, 96, 96);
	border-style: outset;
	border-radius: 15px;
	border-image: url(:/MainWindow/Media/house.png);
}
QToolButton:hover{
	background-color: rgb(55, 55, 55);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>740</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	background-color: rgb(200, 0, 0);
	border-style: outset;
	border-radius: 25px;
	border-image: url(:/MainWindow/Media/exit.png);
}
QPushButton:hover{
	background-color: rgb(150, 0, 0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
